app/src/main/java/com/worldtreeinc/leaves/activity/LoginActivity.java

```diff
@@ -45,7 +46,10 @@ public void onClick(View v) {
                 case R.id.FacebookLoginButton:
                     userAuthentication.FacebookLogin();
                     break;
-            }
+                case R.id.resetPassword:
+                    Intent resetPassword = new Intent(this, ResetPasswordActivity.class);
```

Comment:
NITPICK: Line 43 and 44 looks similar to this, which is not DRY. Write a single method to start Intents so that you do not repeat yourself.